src/ImageSharp/ColorSpaces/CieLch.cs

```diff
@@ -13,15 +13,15 @@ namespace SixLabors.ImageSharp.ColorSpaces
     /// </summary>
     public readonly struct CieLch : IEquatable<CieLch>
     {
-        private static readonly Vector3 Min = new Vector3(0, -200, 0);
-        private static readonly Vector3 Max = new Vector3(100, 200, 360);
-
         /// <summary>
         /// D50 standard illuminant.
         /// Used when reference white is not specified explicitly.
         /// </summary>
         public static readonly CieXyz DefaultWhitePoint = Illuminants.D50;
 
+        private static readonly Vector3 Min = new Vector3(0, -200, 0);
```

Comment:
Public fields should preceed private fields, right? I always get confused by these rules.

Reply:
Yep (and why I reordered this one).

Reply:
I'm surprised StyleCop hadn't already picked up on this one.